c/Makefile

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (c) Citrix Systems, Inc.
```

Comment:
Does this need to change to "Cloud Software Group Holdings, Inc." now or is this for future when legal tell us what to call ourselves?

Reply:
You're right, the new copyright is to be used for new files and this Makefile is technically new while the other files are transfers from a different repo. I'll correct it.

Reply:
Done.